server/server.js

```diff
@@ -11,6 +12,15 @@ app.use(morgan('tiny'));
 app.use(authentication);
 app.use(express.static(path.join(__dirname, '../client/dist/')));
 
+app.all('/*', (req, res) => {
```

Comment:
You commit in this pull request explains what's the use of app.all function. I think write inside code and commit out about what the function is and examples of what it can realize will be really helpful of other delevolpers understanding. Anyway, it looks really cool.